.github/workflows/create-branch.yaml

```diff
@@ -0,0 +1,51 @@
+name: Reusable - Create Branch
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+    secrets:
+      token:
+        required: true
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Create new branch
+        run: |
+          # Validate input
+          if [ -z "${{ inputs.version }}" ]; then
+            echo "ERROR: version input is empty"
+            exit 1
+          fi
+          
+          VERSION="${{ inputs.version }}"
+          MINOR_VERSION="${VERSION::-2}"
```

Comment:
I think in the other script you used a different approach to strip everything past the last `.`. I feel that it is more robust or at least let's be consistent with techniques